bundles/org.openhab.binding.yioremote/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yioremote"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="yioremotedock">
+		<label>YIOremote Dock Binding Thing</label>
+		<description>YIOremote Dock Binding Thing</description>
+		<channel-groups>
+			<channel-group id="input" typeId="input"/>
+			<channel-group id="output" typeId="output"/>
+		</channel-groups>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Network address of the YIO Remote Dock</label>
+				<description>Network address of the YIO Remote Dock</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="accesstoken" type="text" required="true">
+				<label>Access token of the YIO Remote Dock</label>
+				<description>The authentication token for the access currently 0</description>
+				<default>0</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-group-type id="input">
+		<label>Inputs</label>
+		<description>The channels used for Input</description>
+		<channels>
+			<channel id="receiverswitch" typeId="receiverswitch"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="output">
+		<label>Outputs</label>
+		<description>The channels used for Output</description>
+		<channels>
+			<channel id="status" typeId="status"/>
+		</channels>
+	</channel-group-type>
+	<channel-type id="receiverswitch">
+		<item-type>Switch</item-type>
+		<label>YIO Remote Dock receiverswitch</label>
```

Comment:
You don't need to repeat the binding or Thing name here.
```suggestion
		<label>Receiver Switch</label>
```

Reply:
resolved